examples/python/xds/README.md

```diff
@@ -0,0 +1,65 @@
+gRPC Hostname Example
+=====================
+
+The hostname example is a Hello World server whose response includes its
+hostname. It also supports health and reflection services. This makes it a good
+server to test infrastructure, like load balancing.
+
+The example requires grpc to already be built. You are strongly encouraged
+to check out a git release tag, since there will already be a build of gRPC
+available.
+
+### Run the example
+
+1. Navigate to this directory:
+
+```sh
+cd grpc/examples/python/xds
+```
+
+2. Run the server
+
+```sh
+python server.py
+```
+
+3. Verify the Server
+
+This step is not strictly necessary, but you can use it as a sanity check if
+you'd like. If you don't have it, install
+[`grpcurl`](https://github.com/fullstorydev/grpcurl/releases). This will allow
+you to manually test the service.
```

Comment:
Do you want to advertise your installation script?

Reply:
There's since been some controversy about installer scripts like that. I haven't put it into anything official yet.